Backpack/src/main/java/net/skyscanner/backpack/bottomsheet/BpkBottomSheetBehaviour.kt

```diff
@@ -61,6 +61,23 @@ class BpkBottomSheetBehaviour<V : View>(context: Context, attrs: AttributeSet? =
     )
   }
 
+  override fun onLayoutChild(parent: CoordinatorLayout, child: V, layoutDirection: Int): Boolean {
+    return super.onLayoutChild(parent, child, layoutDirection).apply { child.background = background }
```

Comment:
the background was only set when the state changed, meaning if the state was set initially it might not be applied correctly. This fixes that